.gitlab-ci.yml

```diff
@@ -63,80 +68,129 @@ build_wheel_macos_py39:
      - zsh -e scripts/test.sh --wheel-path=${WHEEL_PATH} --python=${PYTHON}
        --test-package=${TEST_PACKAGE} --requirements=${REQUIREMENTS} --fast
 
-test_py39_coremltools_test:
+test_py39_coremltools_test_intel:
   <<: *test_macos_pkg
   tags:
-    - macos12
+    - macOS13
   dependencies:
-    - build_wheel_macos_py39
+    - build_wheel_macos_py39_intel
   variables:
     WHEEL_PATH: build/dist/*cp39*10_15*
     TEST_PACKAGE: coremltools.test
     PYTHON: "3.9"
     REQUIREMENTS: reqs/test.pip
 
-test_py39_pytorch:
+test_py39_pytorch_intel:
   <<: *test_macos_pkg
   tags:
-    - macos12
+    - macOS13
   dependencies:
-    - build_wheel_macos_py39
+    - build_wheel_macos_py39_intel
   variables:
     PYTHON: "3.9"
     TEST_PACKAGE: coremltools.converters.mil.frontend.torch
     WHEEL_PATH: build/dist/*cp39*10_15*
     REQUIREMENTS: reqs/test.pip
 
-test_py37_tf1:
+test_py37_tf1_intel:
   <<: *test_macos_pkg
   tags:
-    - macos12
+    - macOS13
   dependencies:
-    - build_wheel_macos_py37
+    - build_wheel_macos_py37_intel
   variables:
     PYTHON: "3.7"
     TEST_PACKAGE: coremltools.converters.mil.frontend.tensorflow
     WHEEL_PATH: build/dist/*cp37*10_15*
     REQUIREMENTS: reqs/test_tf1.pip
 
-test_py39_tf2:
+test_py39_tf2_intel:
   <<: *test_macos_pkg
   tags:
-    - macos12
+    - macOS13
   dependencies:
-    - build_wheel_macos_py39
+    - build_wheel_macos_py39_intel
   variables:
     PYTHON: "3.9"
     TEST_PACKAGE: coremltools.converters.mil.frontend.tensorflow2
     WHEEL_PATH: build/dist/*cp39*10_15*
     REQUIREMENTS: reqs/test.pip
 
-test_py39_mil:
+test_py39_mil_intel:
   <<: *test_macos_pkg
   tags:
-    - macos12
+    - macOS13
   dependencies:
-    - build_wheel_macos_py39
+    - build_wheel_macos_py39_intel
   variables:
     PYTHON: "3.9"
     TEST_PACKAGE: coremltools.converters.mil.mil
     WHEEL_PATH: build/dist/*cp39*10_15*
     REQUIREMENTS: reqs/test.pip
 
+test_py10_coremltools_test:
```

Comment:
`py310` instead of `py10`

Reply:
That is a typo. Thank. Good catch. I've fixed it.